python/ray/experimental/state/common.py

```diff
@@ -75,6 +76,43 @@ class SummaryResource(Enum):
 PredicateType = str  # Literal["=", "!="]
 
 
+class Humanify:
+    """A class containing default methods to
+    convert units into a human readable string."""
+
+    def timestamp(x: float):
+        """Converts miliseconds to a datetime object."""
+        return str(datetime.datetime.fromtimestamp(x / 1000))
+
+    def memory(x: int):
+        """Converts raw bytes to a human readable memory size."""
+        if x >= 2**30:
```

Comment:
nit: let's keep the number of decimal places as well.

Reply:
Sure!